src/decisions.jl

```diff
@@ -159,7 +159,7 @@ function apply_policies!(x, test)
             # We must do this in the order of policy range
             if haskey(municipality.policies, typeof(intervention))
                 targets = keys(municipality.policies[typeof(intervention)])
-                assign = zip(targets, Iterators.take(xs, length(targets)))
+                assign = collect(zip(targets, Iterators.take(xs, length(targets))))
```

Comment:
This was weird. Last value would always fail in this context if assign was not collected before running the map. It's not that big of a deal, just odd though.